### ISSUE TITLE
Prompt Override stopped working correctly in @2023-05-01 version

### ISSUE THREAD

**Issue by acomas1** [Azure-Samples/azure-search-openai-demo#357]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ x] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> enter into prompt override something like
>>>answer like a pirate
or 
>>>Generate HTML for any formatting request such as summarize in a table, list, highlight, etc.

### Any log messages given by the failure
> NONE

### Expected/desired behavior
> for a pirate - something like 'arrg...'
for table summary - not formatting in HTML

### OS and Version?
> Windows 11

### azd version?
azd version 1.0.2 

### Versions
> Previous 2022 version worked correctly

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by josuakiviranta** [comment]
I also started getting completely incorrect answers when I wrote '''Answer like a pirate''' after updating to the @2023-05-01 version. However, if I now include the prompt override like this: '''>>>> Answer like a pirate''' it seems to work for some reason.

I received a tip from this video: https://www.youtube.com/watch?v=tW2EA4aZ_YQ (at 8:51).

**Comment by PTTrazavi** [comment]
how do you guys use @2023-05-01 version?
I only see version @2023-05-15 in this web page [(Azure OpenAI)](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference)

**Comment by pamelafox** [comment]
I discovered we had a bug which caused us to ignore the prompt override in the chat tab, I just merged a fix for it in pr #606
